Documentation/packaging.md

```diff
@@ -17,6 +17,20 @@ It covers dependencies, file ownership and permissions, and tips to observe pack
 - git
 - gpg
 
+**Fedora 23**
+
+```
+dnf install squashfs-tools glibc-static trousers-devel  @development-tools
+```
```

Comment:
I was thinking of `packaging.md` as a documentation for packagers rather than for developers. Should it go to [hacking.md](https://github.com/coreos/rkt/blob/master/Documentation/hacking.md) instead?

I think that some deps are missing, see [fedora.cloudinit](https://github.com/coreos/rkt/blob/master/tests/cloudinit/fedora.cloudinit#L21)